### ISSUE TITLE
can't unsubscribe

### ISSUE THREAD

**Issue by nikoss** [TeamNewPipe/NewPipe#3811]
### Version

- 0.19.5 and latest versions; this bug might be a year old

### Steps to reproduce the bug

Subscribe to a youtube or peertube channel.
Sometimes later, maybe a week, unsubscribe from that channel.
The channel's newest videos keep popping up in your news feed.
Export the feeds.
Check the exported file with a text editor.
The unsubscribed feed is still there.
Check the GUI of NewPipe.
It shows as if you are not subscribed to the channel.

This bug is quite old. But lately, after I subscribed to a busy peertube channel, it has become annoying. ... because I can not unsubscribe.

I have not tried to purge any cached data (searches, playing points, etc).

edit:
I have this problem, for maybe about a year, with at least one youtube channel.

### Expected behavior

Unsubscribe from the  peertube or youtube channel.

### Actual behaviour

You are still subscribed to the channel

**Comment by Stypox** [comment]
Are you able to unsubscribe if you long-press a channel in the subscription tab and then tap "Unsubscribe"?

**Comment by nikoss** [comment]
I just tried that.
This youtube channel appears on the subscription list.
But if I click on it, the orange button says "subscribe", because I had tried to unsubscribe in the past.
But I still get the channel's videos on my news feed, from time to time.

I now tried, as you said above, and an error with a report came up.
The following is the error log.

___ Logs ___
{"user_action":"requested feed","request":"Loading feed","content_language":"RS","service":"none","package":"org.schabi.newpipe","version":"0.19.5","os":"Linux Android 7.0 - 24","time":"2020-06-28 21:19","exceptions":["org.schabi.newpipe.local.feed.service.FeedLoadService$RequestException: 0:https://www.youtube.com/channel/UCM-wZev_mfdnpGn14byGMYw\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:223)\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:66)\n\tat io.reactivex.internal.operators.parallel.ParallelMap$ParallelMapSubscriber.onNext(ParallelMap.java:113)\n\tat io.reactivex.internal.operators.parallel.ParallelFilter$ParallelFilterSubscriber.tryOnNext(ParallelFilter.java:130)\n\tat io.reactivex.internal.operators.parallel.ParallelRunOn$RunOnConditionalSubscriber.run(ParallelRunOn.java:397)\n\tat io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)\n\tat io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:237)\n\tat java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)\n\tat java.lang.Thread.run(Thread.java:761)\nCaused by: java.lang.RuntimeException: org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Got error: \"This account has been terminated because it is linked to an account that received multiple third-party claims of copyright infringement.\"\n\tat io.reactivex.internal.util.ExceptionHelper.wrapOrThrow(ExceptionHelper.java:46)\n\tat io.reactivex.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:93)\n\tat io.reactivex.Single.blockingGet(Single.java:2870)\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:217)\n\t... 11 more\nCaused by: org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Got error: \"This account has been terminated because it is linked to an account that received multiple third-party claims of copyright infringement.\"\n\tat org.schabi.newpipe.extractor.services.youtube.YoutubeParsingHelper.defaultAlertsCheck(YoutubeParsingHelper.java:501)\n\tat org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeChannelExtractor.onFetchPage(YoutubeChannelExtractor.java:104)\n\tat org.schabi.newpipe.extractor.Extractor.fetchPage(Extractor.java:56)\n\tat org.schabi.newpipe.extractor.channel.ChannelInfo.getInfo(ChannelInfo.java:46)\n\tat org.schabi.newpipe.util.ExtractorHelper.lambda$getChannelInfo$4(ExtractorHelper.java:124)\n\tat org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$u5W7VszTe8AoEexIsFM9huQfbkM.call(lambda)\n\tat io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)\n\tat io.reactivex.Single.subscribe(Single.java:3666)\n\tat io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)\n\tat io.reactivex.Single.subscribe(Single.java:3666)\n\tat io.reactivex.Single.blockingGet(Single.java:2869)\n\t... 12 more\n","org.schabi.newpipe.local.feed.service.FeedLoadService$RequestException: 0:https://www.youtube.com/channel/UCGgcTITLsxCc2A6Qqf-Y9Xg\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:223)\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:66)\n\tat io.reactivex.internal.operators.parallel.ParallelMap$ParallelMapSubscriber.onNext(ParallelMap.java:113)\n\tat io.reactivex.internal.operators.parallel.ParallelFilter$ParallelFilterSubscriber.tryOnNext(ParallelFilter.java:130)\n\tat io.reactivex.internal.operators.parallel.ParallelRunOn$RunOnConditionalSubscriber.run(ParallelRunOn.java:397)\n\tat io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)\n\tat io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:237)\n\tat java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)\n\tat java.lang.Thread.run(Thread.java:761)\nCaused by: java.lang.RuntimeException: org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Got error: \"This account has been terminated because we received multiple third-party claims of copyright infringement regarding material the user posted.\"\n\tat io.reactivex.internal.util.ExceptionHelper.wrapOrThrow(ExceptionHelper.java:46)\n\tat io.reactivex.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:93)\n\tat io.reactivex.Single.blockingGet(Single.java:2870)\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:217)\n\t... 11 more\nCaused by: org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Got error: \"This account has been terminated because we received multiple third-party claims of copyright infringement regarding material the user posted.\"\n\tat org.schabi.newpipe.extractor.services.youtube.YoutubeParsingHelper.defaultAlertsCheck(YoutubeParsingHelper.java:501)\n\tat org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeChannelExtractor.onFetchPage(YoutubeChannelExtractor.java:104)\n\tat org.schabi.newpipe.extractor.Extractor.fetchPage(Extractor.java:56)\n\tat org.schabi.newpipe.extractor.channel.ChannelInfo.getInfo(ChannelInfo.java:46)\n\tat org.schabi.newpipe.util.ExtractorHelper.lambda$getChannelInfo$4(ExtractorHelper.java:124)\n\tat org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$u5W7VszTe8AoEexIsFM9huQfbkM.call(lambda)\n\tat io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)\n\tat io.reactivex.Single.subscribe(Single.java:3666)\n\tat io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)\n\tat io.reactivex.Single.subscribe(Single.java:3666)\n\tat io.reactivex.Single.blockingGet(Single.java:2869)\n\t... 12 more\n","org.schabi.newpipe.local.feed.service.FeedLoadService$RequestException: 0:https://www.youtube.com/channel/UCLCy0r28BuC_AOaEkOTrw1g\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:223)\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:66)\n\tat io.reactivex.internal.operators.parallel.ParallelMap$ParallelMapSubscriber.onNext(ParallelMap.java:113)\n\tat io.reactivex.internal.operators.parallel.ParallelFilter$ParallelFilterSubscriber.tryOnNext(ParallelFilter.java:130)\n\tat io.reactivex.internal.operators.parallel.ParallelRunOn$RunOnConditionalSubscriber.run(ParallelRunOn.java:397)\n\tat io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)\n\tat io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:237)\n\tat java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)\n\tat java.lang.Thread.run(Thread.java:761)\nCaused by: java.lang.RuntimeException: org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Got error: \"This channel does not exist.\"\n\tat io.reactivex.internal.util.ExceptionHelper.wrapOrThrow(ExceptionHelper.java:46)\n\tat io.reactivex.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:93)\n\tat io.reactivex.Single.blockingGet(Single.java:2870)\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:217)\n\t... 11 more\nCaused by: org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Got error: \"This channel does not exist.\"\n\tat org.schabi.newpipe.extractor.services.youtube.YoutubeParsingHelper.defaultAlertsCheck(YoutubeParsingHelper.java:501)\n\tat org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeChannelExtractor.onFetchPage(YoutubeChannelExtractor.java:104)\n\tat org.schabi.newpipe.extractor.Extractor.fetchPage(Extractor.java:56)\n\tat org.schabi.newpipe.extractor.channel.ChannelInfo.getInfo(ChannelInfo.java:46)\n\tat org.schabi.newpipe.util.ExtractorHelper.lambda$getChannelInfo$4(ExtractorHelper.java:124)\n\tat org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$u5W7VszTe8AoEexIsFM9huQfbkM.call(lambda)\n\tat io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)\n\tat io.reactivex.Single.subscribe(Single.java:3666)\n\tat io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)\n\tat io.reactivex.Single.subscribe(Single.java:3666)\n\tat io.reactivex.Single.blockingGet(Single.java:2869)\n\t... 12 more\n","org.schabi.newpipe.local.feed.service.FeedLoadService$RequestException: 3:https://open.tube/api/v1/accounts/ryzenk8@open.tube\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:223)\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:66)\n\tat io.reactivex.internal.operators.parallel.ParallelMap$ParallelMapSubscriber.onNext(ParallelMap.java:113)\n\tat io.reactivex.internal.operators.parallel.ParallelFilter$ParallelFilterSubscriber.tryOnNext(ParallelFilter.java:130)\n\tat io.reactivex.internal.operators.parallel.ParallelRunOn$RunOnConditionalSubscriber.run(ParallelRunOn.java:397)\n\tat io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)\n\tat io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:237)\n\tat java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)\n\tat java.lang.Thread.run(Thread.java:761)\nCaused by: java.lang.RuntimeException: org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Account not found\n\tat io.reactivex.internal.util.ExceptionHelper.wrapOrThrow(ExceptionHelper.java:46)\n\tat io.reactivex.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:93)\n\tat io.reactivex.Single.blockingGet(Single.java:2870)\n\tat org.schabi.newpipe.local.feed.service.FeedLoadService$startLoading$7.apply(FeedLoadService.kt:217)\n\t... 11 more\nCaused by: org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Account not found\n\tat org.schabi.newpipe.extractor.services.peertube.PeertubeParsingHelper.validate(PeertubeParsingHelper.java:29)\n\tat org.schabi.newpipe.extractor.services.peertube.extractors.PeertubeAccountExtractor.getPage(PeertubeAccountExtractor.java:133)\n\tat org.schabi.newpipe.extractor.services.peertube.extractors.PeertubeAccountExtractor.onFetchPage(PeertubeAccountExtractor.java:152)\n\tat org.schabi.newpipe.extractor.Extractor.fetchPage(Extractor.java:56)\n\tat org.schabi.newpipe.extractor.channel.ChannelInfo.getInfo(ChannelInfo.java:46)\n\tat org.schabi.newpipe.util.ExtractorHelper.lambda$getChannelInfo$4(ExtractorHelper.java:124)\n\tat org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$u5W7VszTe8AoEexIsFM9huQfbkM.call(lambda)\n\tat io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)\n\tat io.reactivex.Single.subscribe(Single.java:3666)\n\tat io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)\n\tat io.reactivex.Single.subscribe(Single.java:3666)\n\tat io.reactivex.Single.blockingGet(Single.java:2869)\n\t... 12 more\n"],"user_comment":""}

**Comment by Stypox** [comment]
The report says "This account has been terminated because it is linked to an account that received multiple third-party claims of copyright infringement." for the channel https://www.youtube.com/channel/UCM-wZev_mfdnpGn14byGMYw

Was that the channel you just unsubscribed from?

**Comment by nikoss** [comment]
The channel is...
https://www.youtube.com/user/Zedman5

Now I exported the Subscribed Channels List.
The channel has successfully been removed.
So has the PeerTube channel that I was trying to unsubscribe from (now, after I have used the subscriptions list "method").
Therefore unsubscribing by using the Subscriptions List (tab) works!
Despite the error message.

But this is not how I was trying to unsubscribe.
- I was going to the feed list (tab)
- I was clicking on the "unwanted" video
- Then clicked on the publisher's name (or is it channel ?)
- Then clicked on the subscription button
- The button changed, showing as if I was no longer subscribed
- But, as I have described in the beginning of this issue, in fact, I was still subscribed to the channel (or publisher)
- I didn't know, of course, that there was at least one more way to unsuscribe
